mautic/mautic
# Salesforce company push

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | Pending
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4853
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Companies Push to Salesforce. this PR adds the ability to push companies to Salesforce in the CLI command.
[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply this PR.
2. if you have Accounts selected for Salesforce plugin, using the command `php app/console mautic:integration:fetchleads  --integration=Salesforce` will now push companies too. 
3. check that companies are not duplicated, and updated properly.
